js/well_of_ideas.test.js

```diff
@@ -0,0 +1,18 @@
+const well = require('./well_of_ideas');
+
+
+describe('well_of_ideas', () => {
+
+  test('should be Fail', () => {
```

Comment:
Let's describe _when_ it should be _Fail_, not only that it should have this value

The same applies to other tests